src/teststat/ChiSquare.h

```diff
@@ -15,6 +15,7 @@ class DataSet;
 class ChiSquare : public TestStatistic{
  public:
     ChiSquare() : fCalculatedDataPdf(false) {}
+    ~ChiSquare();
```

Comment:
the default destructor should be fine for this class, there's no memory management